### ISSUE TITLE
Declaration containing NTP servers by dns name failing

### ISSUE THREAD

**Issue by kwinkelman-bw** [F5Networks/f5-declarative-onboarding#125]
### Environment
 * Declarative Onboarding Version: 1.10.0
 * BIG-IP Version: 13.1.3.1 Build 0.0.4 Point Release 1 

### Summary
Declaration containing NTP servers by dns name failing with "Unable to resolve host: <hostname> getaddrinfo EAI_AGAIN". Applying initial configuration after deploying new VE.

### Steps To Reproduce
1. Submit the following declaration:
```json
{
            "$schema": "https://raw.githubusercontent.com/F5Networks/f5-declarative-onboarding/master/src/schema/1.9.0/base.schema.json",
            "schemaVersion": "1.9.0",
            "class": "Device",
            "async": true,
            "label": "My first onboarding, so cute",
            "Common": {
                "class": "Tenant",
                "mySystem": {
                    "class": "System",
                    "hostname": "***",
                    "consoleInactivityTimeout": 0,
                    "cliInactivityTimeout": 0
                },
                "myLicense": {
                    "class": "License",
                    "licenseType": "regKey",
                    "regKey": "***",
                    "overwrite": false
                },
                "myDns": {
                    "class": "DNS",
                    "nameServers": [
                        "192.168.101.114",
                        "192.168.101.113"
                    ],
                    "search": [
                        "***",
                        "***",
                        "***",
                        "***"
                    ]
                },
                "myNtp": {
                    "class": "NTP",
                    "servers": [
                        "idm-01a.lab1.<domain>.local",
                        "idm-01b.lab1.<domain>.local"
                    ],
                    "timezone": "UTC"
                }
            }
        }
```

2. Observe the following error response:
```json
    {
        "id": "10124329-2426-47ef-8dee-860ac2cc2832",
        "selfLink": "https://localhost/mgmt/shared/declarative-onboarding/task/10124329-2426-47ef-8dee-860ac2cc2832",
        "code": 424,
        "status": "ERROR",
        "message": "invalid config - rolled back",
        "errors": [
            "tryUntil: max tries reached: Unable to resolve host idm-01a.lab1.<domain>.local: getaddrinfo EAI_AGAIN",
            "tryUntil: max tries reached: Unable to resolve host idm-01a.lab1.<domain>.local: getaddrinfo EAI_AGAIN"
        ],
        "result": {
            "class": "Result",
            "code": 424,
            "status": "ERROR",
            "message": "invalid config - rolled back",
            "errors": [
                "tryUntil: max tries reached: Unable to resolve host idm-01a.lab1.<domain>.local: getaddrinfo EAI_AGAIN",
                "tryUntil: max tries reached: Unable to resolve host idm-01a.lab1.<domain>.local: getaddrinfo EAI_AGAIN"
            ]
        },
    "declaration": {}
}
```

### Expected Behavior
Declaration succeeds and applies configuration.

### Actual Behavior
Declaration is not applied, and config is rolled back to initial config.

**Comment by kwinkelman-bw** [comment]
[restnoded.log.gz](https://github.com/F5Networks/f5-declarative-onboarding/files/4252390/restnoded.log.gz)

**Comment by kwinkelman-bw** [comment]
DNS resolution is working with a simple dig from the VE:`[root@lb-testing-01a:Active:Standalone] restnoded # dig @192.168.101.114 idm-01a.<REDACTED>.local +short
192.168.101.113`

**Comment by dstokesf5** [comment]
Confirmed and added to our internal product backlog as AUTOTOOL-2041.

**Comment by dstokesf5** [comment]
Actually it looks like this one was not confirmed after all. We are having some difficulties with reproducing this one. Are you still seeing it on the most recent version of Declarative Onboarding?

**Comment by dstokesf5** [comment]
This issue was resolved as part of the 1.21.0 release.
